app/code/Magento/MediaContentSynchronization/Console/Command/Synchronize.php

```diff
@@ -58,12 +48,7 @@ protected function configure()
     protected function execute(InputInterface $input, OutputInterface $output)
     {
         $output->writeln('Synchronizing content with assets...');
-        $this->state->emulateAreaCode(
```

Comment:
This emulation is used for Magento to generate proper links when searching for content usages. When removing the emulation - the links generation should be fixed